include/sqlpp11/mysql/connection.h

```diff
@@ -294,6 +294,8 @@ namespace sqlpp
 
       //! report a rollback failure (will be called by transactions in case of a rollback failure in the destructor)
       void report_rollback_failure(const std::string message) noexcept;
+      
+      detail::connection_handle_t* get_handle();
```

Comment:
`connection_handle_t` is an implementation detail. You wanted to get the MYSQL handle. You should return that.